.ci/test_lint_doctests.py

```diff
@@ -81,6 +82,36 @@ def test_docker_build_matrix():
         assert existing_build_matrix == f.read()
 
 
+@pytest.mark.timeout(30)
+def test_json_schemas():
```

Comment:
In addition to this check that validates the json schemas are up to date, can we use a pre-commit hook to validate the yamls all pass the jsonschema check? See https://github.com/python-jsonschema/check-jsonschema